tripy/tripy/frontend/trace/ops/binary_elementwise.py

```diff
@@ -144,6 +145,39 @@ def to_flat_ir(self, inputs, outputs):
             DivideOp.build(inputs, [divide_out])
             # Then apply FloorOp to the result of the division
             FloorOp.build([divide_out], outputs)
+        elif self.kind == BinaryElementwise.Kind.MOD:
```

Comment:
We can use `stablehlo.remainder`: https://github.com/openxla/stablehlo/blob/main/docs/spec.md#remainder

Reply:
@yizhuoz004 remainder and modulo are not the same for negative numbers and I think adding conditionals for checking negative number and adjusting for that will be compute intensive compared to the current implementation.

Reply:
I see, it confuses me that stablehlo has `remainder` but not `modulo` which is more common...